honeybee_radiance/sensor.py

```diff
@@ -92,9 +156,6 @@ def __eq__(self, value):
     def __ne__(self, value):
         return not self.__eq__(value)
 
-    def ToString(self):
```

Comment:
Why are we deleting `ToString` here?

Reply:
This was a redefinition of `ToString`. You'll see that it already exists on the class if you scroll up.